### PR TITLE
Remove terra-base class

### LINKED ISSUE #490: Base component add terra-Base class on html tag
# Issue Description
We discussed in a meeting the other day about having the base component automatically add the class `terra-Base` on the html tag.

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Resolves #490 

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
With this change do we want to remove `htmlTag.setAttribute('class', 'terra-Base');` from the jestsetup.js file?

File: packages/terra-site/src/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html class="terra-Base" lang="en-US" dir="ltr">
+<html lang="en-US" dir="ltr">
```
Comment (Matt-Butler) [inline packages/terra-site/src/index.html]: here too:
https://github.com/cerner/terra-core/blob/a4a5230c19ce5951832863cb0c0e9b6443459119/packages/terra-toolkit/tests/nightwatch/fixtures/index.html

Comment (bjankord) [inline packages/terra-site/src/index.html]: Updated: https://github.com/cerner/terra-core/pull/506/commits/6287d46c1a0fd7403e37f07a1211f19f1daab610

**Comment by bjankord** [comment]
@emilyrohrbough
> With this change do we want to remove htmlTag.setAttribute('class', 'terra-Base'); from the jestsetup.js file?

Updated: https://github.com/cerner/terra-core/pull/506/commits/6287d46c1a0fd7403e37f07a1211f19f1daab610

**Pull request merged**